src/main/java/ch/sportchef/business/event/boundary/EventImageResource.java

```diff
@@ -102,6 +109,9 @@ public Response uploadImage(@Context HttpServletRequest request) throws IOExcept
                 }
             }
         }
+        BufferedImage plainImage = ImageIO.read(file);
+        BufferedImage resizedImage = Scalr.resize(plainImage, IMAGE_WIDTH, IMAGE_HEIGHT);
+        ImageIO.write(resizedImage, FILE_EXTENSION, file);
 
         return Response.ok().build();
```

Comment:
With the resizing code this method has now a higher potential of failure. It should not always return `ok` as the status code. Depending on the error this method should return a descriptive status code like `bad request` if there was no image file in the upload or an `internal server error` if the resizing fails.